### PR TITLE
Add documentation for the rescale-by-width plot option

### PR DESCRIPTION
**Pull request by MJKirk**
See #710

### CODE REVIEW

**Comment by MJKirk** [comment]
The xrange option could also be moved to my new subsection of optional parameters in the docstring now.

As well, as far as I can see, the current docstring is slightly wrong, in that it currently says it applies for multivariate constraints
https://github.com/eos/eos/blob/b639be8d351148c34f632c5877dfd89952a1283c/python/eos/plot/plotter.py#L784
while if you look at the code it would seem to apply to any time of constraint (although currently I don't think there are any plain Gaussian constraints which apply to multiple kinematic values which could use this option).

**Comment by mreboud** [comment]
> The xrange option could also be moved to my new subsection of optional parameters in the docstring now.
> 
> As well, as far as I can see, the current docstring is slightly wrong, in that it currently says it applies for multivariate constraints
> 
> https://github.com/eos/eos/blob/b639be8d351148c34f632c5877dfd89952a1283c/python/eos/plot/plotter.py#L784
> 
> while if you look at the code it would seem to apply to any time of constraint (although currently I don't think there are any plain Gaussian constraints which apply to multiple kinematic values which could use this option).

I agree that this can be moved to your `optional` section, but I don't understand your second comment. In the case of a single Gaussian constraint, the xrange option would not mean much since there is only one kinematic specification. This option allows you to remove some of the measurements when plotting a constraint with many entries.

**Comment by MJKirk** [comment]
> I don't understand your second comment. In the case of a single Gaussian constraint, the xrange option would not mean much since there is only one kinematic specification. This option allows you to remove some of the measurements when plotting a constraint with many entries.

Yeah, sorry, that is me not fully understanding the Constraints format. I was thinking the Multivariate constraints were just for constraints that  applied to multiple observables, not also multiple kinematic variable bins. So ignore

**Comment by mreboud** [comment]
Well that's perfect, it means that this is not 100% clear and could be clarified somewhere in the documentation!

File: python/eos/plot/plotter.py [hunk 1]
```diff
@@ -789,6 +789,10 @@ class Constraint(BasePlot):
            Must identify one of the observables known to EOS; see `the complete list of observables <../observables.html>`_.
            This is only mandatory in multivariate constraints, since these can constrain more than one observable simultaneously.
 
+        The following keys are optional:
+
+         * ``rescale-by-width`` (*bool*) -- Rescales binned constraints by the bin width. Defaults to false.
```
Comment (dvandyk) [inline python/eos/plot/plotter.py:794]: I think we should be more specific here. Does it rescale by the inverse of the bin width or the bin width? I think it is the former, to turn integrated measurements in averaged measurements, making it possible to show integrated measurements in a differential plot.

**Review by dvandyk — CHANGES_REQUESTED** [review]

File: python/eos/plot/plotter.py (hunk 1 above)
Comment (MJKirk) [reply python/eos/plot/plotter.py]: Better?

Comment (mreboud) [reply python/eos/plot/plotter.py]: Definitely! Can you also add something like: `This is often required to compare theory (integrated) predictions and experimental (averaged) measurements.`

Comment (MJKirk) [reply python/eos/plot/plotter.py]: Done

**Comment by dvandyk** [comment]
@MJKirk Can you squash both commits into one please? We do not need two commits for one one-line change :smile:

**Comment by MJKirk** [comment]
> @MJKirk Can you squash both commits into one please? We do not need two commits for one one-line change 😄

Microsoft charging by the commit now :stuck_out_tongue_winking_eye:?
Sure, will do

**Review by mreboud — APPROVED** [review]

**Pull request merged**